### DISCUSSION TITLE
英無限購債救市 殖利率寫多年最大跌幅

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#728] [General]
最近英國政府的減稅政策，讓債券殖利率升的很高，我甚至有看到其他新聞說，英國現在的殖利率跟一些經濟不好的國家相同，這件事其實讓我很驚訝，因為沒想到市場的反應會這麼大，可能還是跟通膨有關吧
網址：https://tw.stock.yahoo.com/news/美債-英無限購債救市-殖利率寫多年最大跌幅-230800717.html

**Comment by CPCpyt** [comment]
利率升高是近年整體經濟的趨勢，高通膨高利率低成長會是近年來的寫照。
